Makefile

```diff
@@ -57,6 +56,11 @@ git_export: clean
 	# Windows needs a different name for the startup script - due to process creation
 	# (no fork/exec)
 	@cp "$(EXPORT_DIR)/scripts/pycam" "$(EXPORT_DIR)/scripts/pycam-loader.py"
+	# overwrite dynamic version detection with a hardcoded string and update deb changelog
+	cd "$(EXPORT_DIR)"; \
+		echo 'VERSION = "$(VERSION)"' >pycam/version.py; \
+		debchange --newversion "$$(echo "$(VERSION)" | tr - .)" "Upstream build"; \
+		debchange --release ""
```

Comment:
Yes, I was already surprised by your decision to treat it as a native package. This contradicts my understanding of native packages of being primarily targeted at debian-like environments.
Anyway: since this does not seem to cause a problem for now, I do not mind.

Regarding the changelog: I prefer a well-maintained and manually assembled changelog file in the project's root. Thus I do not consider it to be important to maintain a development-oriented debian/changelog, since the deb package will always contain both the upstream changelog and the debian/changelog.

Anyway: I have no strong feelings about all of this.